docs/deployment.rst

```diff
@@ -9,6 +9,59 @@ Caching
 
 There is a plenty of options you can use to create proper caching infrastructure: Varnish, CloudFlare, ...
 
+Cloudflare Cache Rules
+-----------
+
+Cloudflare supports custom rules for configuring caching:
+https://developers.cloudflare.com/cache/about/cache-rules/
+
+tileserver-gl renders tiles in multiple formats - ``.png``, ``.jpg (jpeg)``, ``.webp`` for the raster endpoints, ``.pbf`` for vector endpoint. In addition, style information is generated with ``.json`` format.
+https://maptiler-tileserver.readthedocs.io/en/latest/endpoints.html#rendered-tiles
```

Comment:
Could you do me a favor and just remove this link. 'maptiler-tileserver.readthedocs.io' may be temporary if I can get access to the original url 'tileserver.readthedocs.io' again. So I'd rather not have links to it in the docs

Reply:
Makes sense. Removed the link.